integration-scripts/src/common.ts

```diff
@@ -0,0 +1,63 @@
+import { ethers } from 'ethers'
+import chalk from 'chalk'
+
+/**
+ * Devnet miner address
+ */
+export const DEVNET_ADDRESS = '0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f'
+
+/**
+ * Default credentials for testing node
+ */
+export const credentials = {
+  email: 'notreal@fakeemail.ch',
+  password: 'twochains',
+}
+
+export function createProvider(): ethers.providers.JsonRpcProvider {
+  const port = process.env.ETH_HTTP_PORT || `18545`
```

Comment:
How do you feel about making these `const`s like `DEVNET_ADDRESS`?

Reply:
hm, not sure how I feel about this. I scope it to here since only the function will use it, and try to keep as much as I can out of global scope